velox/dwio/common/Options.h

```diff
@@ -36,6 +36,7 @@
 #include "velox/dwio/common/encryption/Encryption.h"
 #include "velox/type/Timestamp.h"
 #include "velox/type/tz/TimeZoneMap.h"
+#include "velox/vector/arrow/Bridge.h"
```

Comment:
Nit : Why is this include needed ?

Reply:
It's for TimestampUnit type on line 631:
```
// Parquet specific options.
  std::optional<TimestampUnit> parquetWriteTimestampUnit;
 ```

Reply:
Yes, but the type from the config is read as `uint8_t` - it is represented externally in the HiveConfig and read by `readTimestampUnit` from the config? So this can be used and then get converted to the type for the parquet writer later? That way we don't leak the vector/arrow header into common headers.

Reply:
Removed include